app/src/main/java/org/listenbrainz/android/util/Constants.kt

```diff
@@ -35,6 +35,11 @@ object Constants {
         const val REFRESH_TOKEN = "refresh_token"
         const val STATUS_LOGGED_IN = 1
         const val STATUS_LOGGED_OUT = 0
+        const val RECOMMENDATION_GREETING = "Song recommended successfully!"
```

Comment:
@pranavkonidena Please move these to the strings file in res. That would enable us to be able to translate these.

Reply:
@pranavkonidena please remove the strings from Constants and use them directly from res/strings

Reply:
@akshaaatt So sorry , I misunderstood what I needed to do.. hopefully it is fixed now